cluster/get-kube-binaries.sh

```diff
@@ -44,6 +44,10 @@ KUBE_ROOT=$(cd $(dirname "${BASH_SOURCE}")/.. && pwd)
 KUBERNETES_RELEASE_URL="${KUBERNETES_RELEASE_URL:-https://storage.googleapis.com/kubernetes-release/release}"
 
 function detect_kube_release() {
+  if [[ -n "${KUBERNETES_RELEASE:-}" ]]; then
```

Comment:
looking at how `get-kube.sh` is currently implemented, I think we want to `s/KUBERNETES_RELEASE/KUBE_VERSION/g` this file - `get-kube.sh` sets `KUBERNETES_RELEASE` to `release/stable` by default, which isn't going to work here.